devices/ePaper/Drivers/Ssd1681.cs

```diff
@@ -0,0 +1,718 @@
+            AutoWriteRAMForRegularPatternRed = 0x46,
+            AutoWriteRAMForRegularPatternBlackWhite = 0x47,
+            SetRAMAddressCounterX = 0x4e,
+            SetRAMAddressCounterY = 0x4f,
+            NOP = 0x7f
+#pragma warning restore CS1591 // Missing XML comment for publicly visible type or member
+#pragma warning restore SA1602 // The enumeration sub-item must have a documentation header
+        }
+
+        /// <summary>
+        /// SSD1681 Supported Sleep Modes.
+        /// </summary>
+        public enum SleepMode : byte
+        {
+            /// <summary>
+            /// Normal Sleep Mode.
+            /// In this mode: 
+            /// - DC/DC Off 
+            /// - No Clock 
+            /// - No Output load 
+            /// - MCU Interface Access: ON
+            /// - RAM Data Access: ON.
+            /// </summary>
+            Normal = 0x00,
+
+            /// <summary>
+            /// Deep Sleep Mode 1.
+            /// In this mode: 
+            /// - DC/DC Off 
+            /// - No Clock 
+            /// - No Output load 
+            /// - MCU Interface Access: OFF
+            /// - RAM Data Access: ON (RAM contents retained).
+            /// </summary>
+            DeepSleepModeOne = 0x01,
+
+            /// <summary>
+            /// Deep Sleep Mode 2.
+            /// In this mode: 
+            /// - DC/DC Off 
+            /// - No Clock 
+            /// - No Output load 
+            /// - MCU Interface Access: OFF
+            /// - RAM Data Access: OFF (RAM contents NOT retained).
+            /// </summary>
+            DeepSleepModeTwo = 0x11,
+        }
+
+        /// <summary>
+        /// SSD1861 Supported Refresh Modes.
+        /// </summary>
+        public enum RefreshMode : byte
+        {
+            /// <summary>
+            /// Causes the display to perform a full refresh of the panel (Display Mode 1).
+            /// </summary>
+            FullRefresh = 0xf7,
+
+            /// <summary>
+            /// Causes the display to perform a partial refresh of the panel (Display Mode 2).
+            /// </summary>
+            PartialRefresh = 0xff,
+        }
+
+        /// <summary>
+        /// SSD1681 RAM.
+        /// </summary>
+        public enum Ram : byte
+        {
+            /// <summary>
+            /// Specifies the black and white RAM area.
+            /// </summary>
+            BlackWhite = 0x00,
+
+            /// <summary>
+            /// Specifies the Colored RAM area (Red for SSD1681).
+            /// </summary>
+            Color = 0x01,
+        }
+
+#pragma warning restore SA1201 // Missing XML comment for publicly visible type or member
+
+        #endregion
+
+        #region IDisposable
+
+        private void Dispose(bool disposing)
+        {
+            if (!this.disposed)
+            {
+                if (disposing)
+                {
+                    this.resetPin.Dispose();
+                    this.busyPin.Dispose();
```

Comment:
in the pattern where uyou can creat the Gpiocontroller, make sure you dispose it if you create it.
also the SpiDevice may not be disposed if you don't create it
still both should be set to null for the garbage collectore to do his job